instances/latest/contentTypes/application_vnd.nest-desktop.model.v4+json.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+    "@context": {
+      "@vocab": "https://openminds.om-i.org/props/"
+    },
+    "@id": "https://openminds.om-i.org/instances/contentTypes/application_vnd.nest-desktop.model.v4+json",
+    "@type": "https://openminds.om-i.org/types/ContentType",
+    "dataType": null,
+    "description": "NEST Desktop model specification version 4",
```

Comment:
Could we provide a bit more than this. This should not be the same as the synonym